webapi/_test_data/test_route/texts/beol/header.xsl

```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:transform xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+               xmlns:xs="http://www.w3.org/2001/XMLSchema"
+               xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+               xmlns:rdfs1="http://www.w3.org/2000/01/rdf-schema#"
+               xmlns:beol="http://0.0.0.0:3333/ontology/0801/beol/v2#"
+               xmlns:knora-api="http://api.knora.org/ontology/knora-api/v2#"
+               exclude-result-prefixes="rdf beol knora-api xs rdfs1" version="2.0">
+
+    <xsl:output method="xml" omit-xml-declaration="yes" encoding="utf-8" indent="yes"/>
+
+    <!-- make IAF id a URL -->
+    <xsl:function name="knora-api:iaf" as="xs:anyURI">
+        <xsl:param name="input" as="xs:string"/>
+        <xsl:value-of select="replace($input, '\(DE-588\)', 'http://d-nb.info/gnd/')"/>
+    </xsl:function>
+
+    <!-- make a standard date (Gregorian calendar assumed) -->
```

Comment:
Does this date processing belong in a BEOL-specific XSL transformation, or should it be in a generic one?

Reply:
beol specific, it belongs to the test case in the integration tests

Reply:
sorry, I misunderstood

I will add a sentence explaining that people could reuse the XSLT functions.

Reply:
done in https://github.com/dhlab-basel/Knora/pull/862/commits/c836d617ca66a1f635e84221503554abf148128e